src/lib/services/event-announcer-service.ts

```diff
@@ -0,0 +1,22 @@
+import { IUnleashConfig } from '../types/option';
+import { IUnleashStores } from '../types/stores';
+import { Logger } from '../logger';
+import { IEventStore } from '../types/stores/event-store';
+
+export default class EventAnnouncer {
```

Comment:
Now that I see this code it seems like the announcer is not doing anything on top of the event store. I'd consider using event store directly and it taking a role of an event announcer without creating a new abstraction. The variable can event be called eventAnnouncer and assigned to the EventStore instance.

Reply:
Yeah, I thought about that. The reason I didn't is that we're currently doing all our scheduling in `src/lib/services/index.ts`, which only works with services. 

We could potentially do something similar in `src/lib/db/index.ts`, but that would require getting hold of the services, which we currently don't. 

We could look into either of these solutions, but they both break with an established pattern, so I'm not sure that's preferable over having this admittedly anemic service. 

what do you think?

Reply:
given this context I'd go with your original solution